src/main/resources/interface-specification.yml

```diff
@@ -651,12 +651,11 @@ paths:
             type: string
       responses:
         '200':
-          description: EService document retrieved
+          description: EService document metadata retrieved
           content:
-            application/octet-stream:
+            application/json:
               schema:
-                type: string
-                format: binary
+                $ref: '#/components/schemas/EServiceDoc'
```

Comment:
EServiceDoc needs to extend with path information coming from catalog management because BFF uses that information to get files from S3.

Reply:
While working on the same route in the BFF we've spotted this mistake! We solved it right now